libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -378,6 +378,10 @@
 #define __EXTFLASHFUNC__
 #endif
 
+#ifndef __INITFUNC__
```

Comment:
Could we add a comment here to help future developers when they can or cannot use this function?

Reply:
Done